apps/prairielearn/src/pages/error/error.html.ts

```diff
@@ -0,0 +1,129 @@
+import jsonStringifySafe from 'json-stringify-safe';
+import _ from 'lodash';
+
+import { formatErrorStack } from '@prairielearn/error';
+import { HtmlSafeString, html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { formatQueryWithErrorPosition } from '@prairielearn/postgres';
+
+import { config } from '../../lib/config.js';
+
+function formatJson(value: any): string {
+  return jsonStringifySafe(value, null, '    ');
+}
+
+export function ErrorPage({
+  error,
+  errorId,
+  referrer,
+  resLocals,
+}: {
+  error: {
+    message?: string;
+    stack?: string;
+    status?: number;
+    data?: Record<string, any>;
+    info?: HtmlSafeString;
+  };
+  errorId: string;
+  referrer: string | null;
+  resLocals: Record<string, any>;
+}) {
+  const {
+    outputStderr,
+    outputStdout,
+    sql: sqlQuery,
+    sqlParams,
+    sqlError,
+    ...restData
+  } = error.data ?? {};
+
+  const formattedSqlQuery = formatQueryWithErrorPosition(sqlQuery, sqlError?.position);
+
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../partials/head') %>", {
+          ...resLocals,
+          pageTitle: `Error ${error.status}`,
+        })}
+      </head>
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../partials/navbar') %>", {
+          ...resLocals,
+          navPage: 'error',
+        })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-danger text-white">Error processing request</div>
+
+            <div class="card-body">
+              <h4 class="mb-3">${error.message}</h4>
+
+              ${error.info ?? ''}
+
+              <p><strong>Error ID:</strong> <code>${errorId}</code></p>
+
+              <p><strong>Status:</strong> ${error.status}</p>
```

Comment:
In the previous version, the buttons were found above the status. The new version has them after the status. Was this on purpose?

Reply:
Yes, it was.